pkg/notifier/notifier_set.go

```diff
@@ -1,21 +1,16 @@
-package processor
+package notifier
 
 import (
 	"context"
+	"time"
 
-	"github.com/stackrox/rox/central/role/resources"
 	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/notifiers"
-	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stackrox/rox/pkg/sync"
 )
 
-var (
-	integrationSAC = sac.ForResource(resources.Integration)
-)
-
-// NotifierSet is a set that coordinates present policies and notifiers.
-type NotifierSet interface {
+// Set is a set that coordinates present policies and notifiers.
+type Set interface {
```

Comment:
Why not leave the name as NotifierSet? Set is too generic.

Reply:
Drat, this is the same roxcet naming problem (notifier.NotifierSet doesn't fly) right?

Reply:
Style checks didn't like it starting with Notifier when the package is named notifier

Reply:
Resolved in slack conversation - the naming convention Notifier.Set should work